### DISCUSSION TITLE
Problems deploying backpack for Laravel on VPS

### DISCUSSION THREAD

**Discussion by AlonMoshe** [Laravel-Backpack/community-forum#933] [Q&A (Help)]
I am developing a Backpack for Laravel project locally using wamp, visual studio code, and git. In the local environment everything works correctly.

When I try to deploy the Backpack for Laravel project on a VPS, it doesn't work. That is, when I browse to the URL of the project: https://example.com/, I see the Laravel page of a new project. When I browse to https://example.com/admin, I get: "The requested URL was not found on this server", instead of the usual login page of backpack.

These are the steps I followed to deploy the project:

Clone from GitHub

Make relevant changes in .env:

2.1 Change APP_ENV from local to production, 2.2 Change APP_URL, 2.3 Change database credentials

Run migrations

make sure that all the permissions are correct in all directories and subdirectories of the system

composer install

Repeat step 2

**Comment by jcastroa87** [comment]
Hello @AlonMoshe 

What kind of server are you using on VPS? it is Apache, NGINX or any other? are you sure is properly configure? because error 404 look by server side.

Let me know this information + backpack version

```
php artisan backpack:version
```

Cheers.

**Comment by AlonMoshe** [comment]
Hello @jcastroa87 , Thanks for your reply. I am using Apache, and I think it is well configured, because I can see the Laravel page when I navigate to my app_url. The problem is that I cannot get the backpack login when I go to app_url/admin

Output of  ```php artisan backpack:version``` 

### PHP VERSION:
8.3.6

### PHP EXTENSIONS:
Core, date, libxml, openssl, pcre, zlib, filter, hash, json, pcntl, random, readline, Reflection, SPL, session, standard, sockets, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, exif, mysqlnd, PDO, Phar, SimpleXML, sodium, sqlite3, tokenizer, xml, xmlwriter, xsl, mcrypt, mysqli, pdo_mysql, pdo_sqlite, xmlreader, zip, Zend OPcache

### LARAVEL VERSION:
11.3.1.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.3.1
backpack/crud: 6.7.6
backpack/generators: v4.0.5
backpack/theme-tabler: 1.2.8

Many thanks,

Alon.

**Comment by jcastroa87** [comment]
I will close this issue because is opened many times, lets continue the discussion in https://github.com/Laravel-Backpack/CRUD/issues/5496

**Reply by AlonMoshe** [reply]
Sure.
